runtime/unit-test/test_probe.cpp

```diff
@@ -0,0 +1,67 @@
+#include "catch2/catch_test_macros.hpp"
+#include <cstdlib>
+#include <cstring>
+#include <unistd.h>
+
+extern "C" {
+
+uint64_t bpftime_probe_read(uint64_t dst, uint64_t size, uint64_t ptr, uint64_t,
+			    uint64_t);
+uint64_t bpftime_probe_write_user(uint64_t dst, uint64_t src, uint64_t len,
+				  uint64_t, uint64_t);
+
+// prepare for future use
+long bpftime_strncmp(const char *s1, uint64_t s1_sz, const char *s2);
```

Comment:
Maybe you don't need to copy these unused declarations now, I'll sync my fork with your once yours is merged, because my PR still has a long way to go